calculator_simple/src/app_window.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file
+ * @brief   (Declarations) View Layer of a simple Calculator application.
+ * 
+ *
+ * @author     Tommy Vegetables <leafyrabbet@gmail.com>
+ * @date       2020-05-17
+ * @copyright  2020, TenonGarden Productions
+ */
+#ifndef HPP_APP_WINDOW_HPP
+#define HPP_APP_WINDOW_HPP
+
+#include <glibmm/ustring.h>
+
+#include <sigc++/sigc++.h>
+
+#include <gtkmm/window.h>
+#include <gtkmm/box.h>
+#include <gtkmm/drawingarea.h>
+#include <gtkmm/grid.h>
+#include <gtkmm/label.h>
+#include <gtkmm/adjustment.h>
+#include <gtkmm/scale.h>
+#include <gtkmm/button.h>
+
+#include <array>
+
+#include "logic_calculator.hpp"
+
+
+class AppWindow :
+              public Gtk::Window
+{
+   public:
+      AppWindow();
+      virtual ~AppWindow();
+
+      void handle_display_update();
+      bool handle_radix_change(const Gtk::ScrollType & scroll_type, const double & radix_new);
```

Comment:
**Consistent Const** failure 👻 

https://isocpp.org/wiki/faq/const-correctness#const-ref-alt

```suggestion
      bool handle_radix_change(Gtk::ScrollType const & scroll_type, double const & radix_new);
```

Reply:
Resolved in 6b16b80